modyn/tests/supervisor/internal/test_supervisor.py

```diff
@@ -172,163 +114,84 @@ def get(self, some_type, pipeline_id):  # pylint: disable=unused-argument
 @patch("modyn.utils.grpc_connection_established", return_value=True)
 @patch.object(Supervisor, "init_metadata_db", noop_init_metadata_db)
 def get_non_connecting_supervisor(
+    minimal_system_config: ModynConfig,
     test_connection_established,
     test_init_evaluator,
     test_init_trainer_server,
     test_init_storage,
     test_init_selector,
 ) -> Supervisor:
-    supervisor = Supervisor(get_minimal_system_config())
+    supervisor = Supervisor(minimal_system_config)
     supervisor.init_cluster_connection()
 
     return supervisor
 
 
-def test_initialization() -> None:
-    get_non_connecting_supervisor()  # pylint: disable=no-value-for-parameter
-
-
-@patch.object(Supervisor, "__init__", noop_constructor_mock)
-def test__validate_training_options():
```

Comment:
There's no need for those trivial tests, pydantic models cannot even be constructed with violating those constraints.